scripts/dict_audit_symbols.py

```
@@ -0,0 +1,299 @@
+                interval_text = match.group(1)
+                codepoints = set()
+                # Parse each interval: { start, end, offset }
+                for line in interval_text.splitlines():
+                    m = re.search(r"\{\s*(0x[0-9a-fA-F]+)\s*,\s*(0x[0-9a-fA-F]+)\s*,\s*(0x[0-9a-fA-F]+)\s*\}", line)
+                    if m:
+                        start = int(m.group(1), 16)
+                        end = int(m.group(2), 16)
+                        for cp in range(start, end + 1):
+                            codepoints.add(cp)
+                font_support[h_file.stem] = codepoints
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)